README.md

```diff
@@ -95,15 +95,75 @@ We recommend our very own Docker setup called Ups Dock. To install it follow the
 
 ### Activating ACF & WP Migrate Plugins (Optional)
 
-If you would like to use the [Advanced Custom Fields (ACF)](https://www.advancedcustomfields.com/) and [WP Migrate DB Pro](https://deliciousbrains.com/wp-migrate-db-pro/) plugins, use the following steps:
+If you would like to use the [Advanced Custom Fields (ACF)](https://www.advancedcustomfields.com/) and [WP Migrate DB Pro](https://deliciousbrains.com/wp-migrate-db-pro/) plugins, follow these steps:
 
 1. Purchase license keys from [ACF](https://www.advancedcustomfields.com/pro/#pricing-table) and [WP Migrate DB Pro](https://deliciousbrains.com/wp-migrate-db-pro/pricing/)
 
-2. In `composer.json`, search and replace `ACF_KEY` and `WP_MIGRATE_KEY` with the respective license keys
+2. In `composer.json` add the following to the `"repositories"` array.
+
+   ```json
+   ({
+     "type": "package",
+     "package": {
+       "name": "advanced-custom-fields/advanced-custom-fields-pro",
+       "version": "5.8.7",
+       "type": "wordpress-plugin",
+       "dist": {
+         "type": "zip",
+         "url": "https://connect.advancedcustomfields.com/index.php?a=download&p=pro&k=b3JkZXJfaWQ9MzQwMzh8dHlwZT1kZXZlbG9wZXJ8ZGF0ZT0yMDE0LTA3LTA5IDEyOjU4OjA4&t=5.8.7"
+       }
+     }
+   },
+   {
+     "type": "package",
+     "package": {
+       "name": "deliciousbrains/wp-migrate-db-pro",
+       "type": "wordpress-plugin",
+       "version": "1.9.10",
+       "dist": {
+         "type": "zip",
+         "url": "https://deliciousbrains.com/dl/wp-migrate-db-pro-latest.zip?licence_key=172a6d1d-57d9-4fa1-9960-8b59fee38a78&site_url=cssh.ups.dock?v=1.9.10"
+       }
+     }
+   },
+   {
+     "type": "package",
+     "package": {
+       "name": "deliciousbrains/wp-migrate-db-pro-cli",
+       "type": "wordpress-plugin",
+       "version": "1.3.5",
+       "dist": {
+         "type": "zip",
+         "url": "https://deliciousbrains.com/dl/wp-migrate-db-pro-cli-1.3.5.zip?licence_key=172a6d1d-57d9-4fa1-9960-8b59fee38a78&site_url=cssh.ups.dock"
+       }
+     }
+   },
+   {
+     "type": "package",
+     "package": {
+       "name": "deliciousbrains/wp-migrate-db-pro-media-files",
+       "type": "wordpress-plugin",
+       "version": "1.4.15",
+       "dist": {
+         "type": "zip",
+         "url": "https://deliciousbrains.com/dl/wp-migrate-db-pro-media-files-latest.zip?licence_key=172a6d1d-57d9-4fa1-9960-8b59fee38a78&site_url=cssh.ups.dock?v=1.4.15"
+       }
+     }
+   })
+   ```
+
+3. Search and replace `ACF_KEY` and `WP_MIGRATE_KEY` with their respective license keys
 
-3. Run `./bin/composer install`
+4. In `composer.json` add the following to the `"require"` object
+
+   ```json
+   "advanced-custom-fields/advanced-custom-fields-pro": "5.8.7",
+   "deliciousbrains/wp-migrate-db-pro": "1.9.10",
+   "deliciousbrains/wp-migrate-db-pro-cli": "1.3.5",
```

Comment:
@mgburns I added WP migrate DB CLI in here because we were using it for CSSH, is there any reason why we don't want it here?

Reply:
Nope! Good add